.github/workflows/ci.yaml

```diff
@@ -155,6 +151,48 @@ jobs:
           docker build -f docker/pmbgpd/Dockerfile -t pmbgpd:_build .
           docker build -f docker/pmbmpd/Dockerfile -t pmbmpd:_build .
           docker build -f docker/pmtelemetryd/Dockerfile -t pmtelemetryd:_build .
```

Comment:
unrelated potential improvement: maybe you just want to iterate with a for loop and build all these flavors given a list of daemons, rather than all this almost duplicated `docker build` statements

Reply:
Yes, good idea. I will remember for the next CI iteration.